pkg/web/static/js/main.js

```diff
@@ -27,7 +27,7 @@ if (logoutBttn) {
 
 document.body.addEventListener('htmx:responseError', (e) => {
   // Close any open modals.
-  document.querySelector('.modal').close()
+  document.querySelector('.modal')?.close()
```

Comment:
Optional chaining was added here so that an error isn't returned if an element with the `.modal` class isn't found on a page.